src/netflow/templates.rs

```diff
@@ -12,14 +12,14 @@ use super::{
 
 #[allow(clippy::struct_field_names)]
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub(super) struct Template {
-    pub(super) header: Netflow9Header,
-    pub(super) template_id: u16,
-    pub(super) field_count: u16,
-    pub(super) flow_length: u64,
-    pub(super) fields: Vec<(u16, u16)>,
-    pub(super) options_template: bool,
-    pub(super) scope_field_count: usize,
+pub(crate) struct Template {
```

Comment:
Since this structure and fields are only shared within the netflow module, pub(super) seems more appropriate.